src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/domain/entities/search/SearchParameterKey.java

```diff
@@ -10,9 +10,9 @@
     STATE("state"),
     TASK_ID("taskId"),
     TASK_TYPE("taskType"),
+    WORK_TYPE("workType"),
```

Comment:
I spoke to @raghera  about this everything on here should be snake_case, however, we have backwards compatibility for camelcase and snake case in R1 keys but moving forward anything R2+ should be snake case.

```suggestion
    WORK_TYPE("work_type"),
```

Reply:
ok